orchagent/portsorch.cpp

```diff
@@ -563,6 +586,10 @@ PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_wi
 
         m_lagIdAllocator = unique_ptr<LagIdAllocator> (new LagIdAllocator(chassisAppDb));
     }
+
+    auto executor = new ExecutableTimer(m_timer, this, "PORT_STATUS_POLL");
+    Orch::addExecutor(executor);
+    m_timer->start();
```

Comment:
This will be migrated to use the implementation in the following PR, which is for AutoNeg, and it will only activate the poll when the port is admin up, link_training feature is available and is enabled.
i.e.
https://github.com/Azure/sonic-swss/pull/2215